model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -82,6 +84,7 @@ class PackageCurationTest : WordSpec({
             with(curatedPkg.pkg) {
                 id.toCoordinates() shouldBe pkg.id.toCoordinates()
                 declaredLicenses shouldBe curation.data.declaredLicenses
+                declaredAuthors shouldBe curation.data.declaredAuthors
```

Comment:
@oheger-bosch  Why did you put `declaredAuthors` after `decalredLicenses`?  Prefer to have it in front as it's the author who set the license and also aligns vars somewhat alphabetically

Reply:
@tsteenbe, this draft PR is based on #3627 from @bs-ondem. Points related to the data model should be discussed there. In this PR, I have just started with extending concrete package managers to populate the new field.